test/basic-test.html

```diff
@@ -431,6 +437,18 @@
     });
   });
 
+  suite('remove key behavior with null target', function () {
+    setup(function () {
+      container = fixture('NoIronA11yKeysBehavior');
+    });
+    test('add and remove a iron-a11y-keys-behavior', function (done) {
+      var element = document.createElement('x-a11y-basic-keys');
+      element.keyEventTarget = null;
+      Polymer.dom(container).appendChild(element);
+      done();
```

Comment:
Nit: since the test is synchronous, you don't need to have to call the done callback (remove it from here and from line 444)